src/azure-cli/HISTORY.rst

```diff
@@ -10,6 +10,12 @@ Release History
 * [BREAKING] When not specified, the default value for `--start-task-wait-for-success` on `az batch pool create` is now true (was false).
 * [BREAKING] The default value for Scope on AutoUserSpecification is now always Pool (was Task on Windows nodes, Pool on Linux nodes). This argument is not exposed via the commandline, but can be set in the `--json-file` arguments.
 
+**Network**
+
+* az network watcher flow-log: Fixed #8132. Support `--interval` to set traffic analysis interval value.
+* az network application-gateway identity: Fixes #10073 and #8244 Add support for setting identity in application-gateway.
+* az network application-gateway ssl-cert: Fixes #8244. Add support for setting key vault id in application-gateway ssl-cert.
```

Comment:
Can you use `fix` in your description not `fixed` or `fixes`? Just in consistent.